packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -16,7 +17,11 @@ import {
 } from '..'
 
 /* Logger */
-const log = new Logger({ name: 'oe:batch-submitter:init' })
+const destination = createWriteStream({
+  dsn: process.env.SENTRY_DSN,
```

Comment:
What happens if this is not set? We need to make sure this works for local development and CI without sending messages to sentry

Reply:
> So far I've tested that Sentry doesn't report errors without a dsn. We won't have pesky alerts, but I've created an issue here to add environment management so we can do away with running these monitoring services at all during localdev: #491

Sorry I missed this comment, if its not set and its a noop then its all good